engine/src/assets/asset.cpp

```diff
@@ -40,17 +40,21 @@ AnyAsset::AnyAsset(std::string_view str)
 {
     if (auto id = uuids::uuid::from_string(str))
     {
-        reflectedId = id.value();
+        pathOrId = str;
         mId = id.value();
     }
+    else if (str.find('/') != std::string_view::npos || str.find('\\') != std::string_view::npos)
```

Comment:
```suggestion
    else if (!str.empty() && str[0] == '/')
```
Paths in the virtual filesystem always use `/`, and they must also start with `/` in this case, so you can do something like this ^